src/aig-common/modules/standard/components/cpv-table/cpv-table.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+    selector: 'aig-cpv-table',
+    templateUrl: './cpv-table.component.html',
+    styleUrls: ['./cpv-table.component.scss']
+})
+export class AigCpvTableComponent implements OnInit {
+    @Input() displayedColumns: string[];
+    @Input() dataSource: any[];
+    
+    testDataSource: any[];
+
+    constructor(
+        private router: Router,
+    ) { }
+
+    ngOnInit(): void {
+        // const test = {
+        //     id: 1,
+        //     name: 'test',
+        //     cpv: 'test'
+        // }
+        // this.testDataSource = [];
+        // this.testDataSource.push(test);
+    }
+
+    public detailCpv(idCpv: string){
```

Comment:
not use a functions for navigate but in html use this:
`[routerLink]="'/s6d/cpv/detail/' + element.id"`